packages/http/fetch/src/observabilityOptions.ts

```diff
@@ -0,0 +1,37 @@
+import { RequestOption } from "@microsoft/kiota-abstractions";
+
+export interface ObservabilityOptions {
+	includeEUIIAttributes: boolean;
+}
+export interface ObservabilityOptionsInternal {
```

Comment:
Add summary for the `ObservabilityOptionsInternal`  and `ObservabilityOptions` especially stating the distinction

Reply:
done